File: exercises-python/2. Problemas de estructura condicional/20.py
'''
20.Desarrolle el programa que lea tres números a, b, c y determine si los números
fueron ingresados en orden ascendente, descendente o en desorden.
'''

import os
os.system("cls")

num1 = int(input("Número 1:"))
num2 = int(input("Número 2:"))
num3 = int(input("Número 3:"))

if num1 == num2 - 1 == num3 - 2: print("Ascendente")
elif num1 == num2 + 1 == num3 + 2: print("Descendente")
else: print("Desordenado: ")

File: exercises-cpp/2. Condicionales/20.cpp
#include <iostream>
#include <cstdio>
using namespace std;

int main(){
    system("cls");

    float num1,num2,num3;

    printf("Numero 1: "); cin>>num1;
    printf("Numero 2: "); cin>>num2;
    printf("Numero 3: "); cin>>num3;
    
    if (num1 == num2 - 1 == num3 - 2) printf("Ascendente");
    else if (num1 == num2 + 1 == num3 + 2) printf("Descendente");
    else printf("Desordenado: ");
    }